src/framework/ui/radio/radio.component.tsx

```diff
@@ -46,54 +54,73 @@ export class Radio extends React.Component<Props> {
   };
 
   private getComponentStyle = (style: StyleType): StyleType => {
-    const { select, highlight, ...container } = style;
+    const { text, select, highlight, ...container } = style;
 
     return {
-      container: {
-        ...container,
-        ...styles.container,
-      },
-      select: {
-        ...select,
-        ...styles.select,
-      },
-      highlight: {
-        ...highlight,
-        ...styles.highlight,
-      },
+      selectContainer: container,
+      select: select,
+      highlight: highlight,
+      text: text,
     };
   };
 
+  private renderTextElement = (style: StyleType): React.ReactElement<TextProps> => {
+    const { text } = this.props;
+
+    return (
+      <Text style={[style, styles.text]} key={0}>{text}</Text>
+    );
+  };
+
+  private renderComponentChildren = (style: StyleType): React.ReactNode => {
+    const { text } = this.props;
+
+    return [
+      text ? this.renderTextElement(style.text) : undefined,
```

Comment:
Should we check also the "text.length" in this case? Will margins from mappings be applied if an empty string is sent?